collectors/perf.go

```diff
@@ -4,12 +4,20 @@
 package collectors
 
 import (
+	"flag"
+	"fmt"
 	"log"
+	"regexp"
 
 	"github.com/StackExchange/wmi"
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+var (
+	volumeWhitelist = flag.String("collector.perf.volume-whitelist", ".+", "Regexp of volumes to whitelist. Volume name must both match whitelist and not match blacklist to be included.")
+	volumeBlacklist = flag.String("collector.perf.volume-blacklist", "_Total", "Regexp of volumes to blacklist. Volume name must both match whitelist and not match blacklist to be included.")
```

Comment:
I'd hardcode the _Total in the code itself, so that a user doesn't have to remember to explicitly exclude it.